openedx/core/djangoapps/content_tagging/rest_api/v1/tests/test_views.py

```diff
@@ -735,61 +1164,66 @@ def test_tag_course(self, user_attr, taxonomy_attr, tag_values, expected_status)
             assert set(t["value"] for t in response.data) == set(tag_values)
 
```

Comment:
@rpenido 

Subjective comment: you could split `test_tag_course` into two test methods, one for the successful `staffA` and `staff` users, and one for the forbidden `user` attempts. That would reduce the number of ddt parameters required, and negate the need for this `if status.is_success` clause.

But objectively: I still don't see any tests for the `GET` ObjectTag endpoint in this file? You could add `get` view tests after these successful `PUT`s like this:
```suggestion
            # Check that re-fetching the tags returns what we set
            response = self.client.get(url, format="json")
            assert status.is_success(response.status_code)
            assert set(t["value"] for t in response.data) == set(tag_values)
```

Reply:
Done 9695d67a30b0222a430226b4c8dab2c4a135df4c